docker/build-tools/Dockerfile

```diff
@@ -37,51 +37,51 @@ ARG TARGETARCH
 ARG ISTIO_TOOLS_SHA
 
 # Pinned versions of stuff we pull in, keep this list sorted alphabetically
-ENV APKO_VERSION=v0.20.1
+ENV APKO_VERSION=v0.22.4
 ENV BENCHSTAT_VERSION=9c9101da8316
 ENV BOM_VERSION=v0.6.0
-ENV BUF_VERSION=v1.47.2
+ENV BUF_VERSION=v1.48.0
 ENV COSIGN_VERSION=v2.4.1
 ENV CRANE_VERSION=v0.20.2
 ENV GCLOUD_VERSION=496.0.0
 ENV GCR_AUTH_VERSION=2.1.21
-ENV GH_VERSION=2.63.2
+ENV GH_VERSION=2.64.0
 ENV GOCOVMERGE_VERSION=b5bfa59ec0adc420475f97f89b58045c721d761c
 ENV GOIMPORTS_VERSION=v0.24.0
 # When updating the golangci version, you may want to update the common-files config/.golangci* files as well.
-ENV GOLANGCI_LINT_VERSION=v1.62.2
+ENV GOLANGCI_LINT_VERSION=v1.63.1
 ENV GOLANG_GRPC_PROTOBUF_VERSION=v1.5.1
-ENV GOLANG_PROTOBUF_VERSION=v1.35.2
+ENV GOLANG_PROTOBUF_VERSION=v1.36.1
 # From Nov 21, 2024. The latest tagged version at the time (0.6.0) is missing
 # some critical bug fixes.
 ENV VT_PROTOBUF_VERSION=79df5c4772f27b4e08f9612d045e1e1b21ac963a
 ENV GO_BINDATA_VERSION=v3.1.2
 ENV GO_JUNIT_REPORT_VERSION=df0ed838addb0fa189c4d76ad4657f6007a5811c
 ENV HADOLINT_VERSION=v2.12.0
-ENV HELM3_VERSION=v3.16.3
-ENV HUGO_VERSION=0.139.3
+ENV HELM3_VERSION=v3.16.4
+ENV HUGO_VERSION=0.140.2
 ENV JB_VERSION=v0.3.1
 ENV JSONNET_VERSION=v0.20.0
 ENV JUNIT_MERGER_VERSION=adf1545b49509db1f83c49d1de90bbcb235642a8
 ENV K8S_CODE_GENERATOR_VERSION=1.29.4
-# From August 18, 2024
-ENV K8S_TEST_INFRA_VERSION=4f5d74517ae8ec387568f8c766f4b0ba1f454129
-# From August 18, 2024
-ENV K8S_PROW_VERSION=2b7da0bc5a15985da67cb951141ca073098660d5
-ENV KIND_VERSION=v0.25.0
+# From Dec 16, 2024
+ENV K8S_TEST_INFRA_VERSION=782e06009fcb315e34c443dbffa4f21136c6f21a
+# From Dec 16, 2024
+ENV K8S_PROW_VERSION=9b059e106dd942101955438369e5dca8afbef2ec
+ENV KIND_VERSION=v0.26.0
```

Comment:
@howardjohn new version of kind usually means we need to push new k8s images, right?

Reply:
You don't need to. Sometimes they make changes to how they are build but its almost always backwards compatible except in a few cases. This one seems fine?